docs/guides/installing-the-slack-cli-for-mac-and-linux.md

```diff
@@ -23,10 +23,10 @@ If you attempt to log in with an older version, you'll receive a `cli_update_req
 **Run the automated installer from your terminal window:**
 
 ```zsh
-curl -fsSL https://downloads.slack-edge.com/slack-cli/install.sh | bash
+curl -fsSL https://downloads.slack-edge.com/tools/slack-cli/install.sh | bash
```

Comment:
I don't think we want to change this URL because our download URL is not changing.